src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -132,8 +132,29 @@ trait IterableOps[+A] extends Any {
   }
 
   override def toString = s"$className(${mkString(", ")})"
+
+  def sameElements[B >: A](that: Iterable[B]): Boolean = {
```

Comment:
We could add a separate trait for collections with stable traversal order that can be extended by both `Seq` and `Sorted`.